cns/restserver/internalapi.go

```diff
@@ -191,6 +196,13 @@
 		}
 	}
 
+	ctx := context.Background()
```

Comment:
+1 on Ramiro's feedback. This should be called from main only after Reconcile is completed. It could be possible that reconcile api is skipped if CRD is not available but IPAM pool monitor should still start.